lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -39,6 +47,15 @@ external set windowFlutterCanvasKit(CanvasKit? value);
 @JS('window.flutterCanvasKit')
 external CanvasKit? get windowFlutterCanvasKit;
 
+@JS('window.h5vcc')
+external H5vcc? get h5vcc;
```

Comment:
I know that we can't yet run tests in Cobalt itself. Would it be possible to craft a Chrome-only test where we should make Chrome pretend that it's Cobalt by monkey-patching `window.h5vcc` and `canvasKit.getH5vccSkSurface` so we could test Cobalt-specific code paths?

Reply:
Added tests as suggested. PTAL.